components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -48,6 +48,7 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
 
   public isLoading = true;
   public saving = false;
+  public attributeList: any;
```

Comment:
Please use strong typing:
```suggestion
  public attributeList: KVPair[];
```